tests/integration_tests/data_steward/tools/create_update_drc_id_match_table_test.py

```diff
@@ -0,0 +1,136 @@
+"""
+Integration test for the create_update_drc_id_match_table module.
+
+Ensures that a partitioned (by hour) drc validation table is created for the site properly and that the created table is
+    populated with 'missing_rdr' if there does not exist data for that field in the `ps_values` table or the default
+    value 'missing_ehr'
+
+Original Issue: DC-1216
+
+The intent of this module is to check that the drc validation table is created properly and the drc validation table is
+ updated properly.
+"""
+
+# Python imports
+import os
+import mock
+from unittest import TestCase
+
+# Third party imports
+from google.cloud.bigquery import DatasetReference, SchemaField
+
+# Project imports
+from utils import bq
+from common import JINJA_ENV
+from app_identity import PROJECT_ID
+from tools import create_update_drc_id_match_table as id_validation
+
+POPULATE_PS_VALUES = JINJA_ENV.from_string("""
+INSERT INTO `{{project_id}}.{{drc_dataset_id}}.{{ps_values_table_id}}` 
+(person_id, first_name, last_name)
+VALUES 
+    (1, 'fee', 'faa'),
+    (2, null, 'foe'),
+    (3, 'fum', null),
+    (4, null, null)
+""")
+
+CONTENT_QUERY = JINJA_ENV.from_string("""
+SELECT *
+FROM {{project_id}}.{{drc_dataset_id}}.{{id_match_table_id}}
+""")
+
+
+class CreateUpdateDrcIdMatchTableTest(TestCase):
+
+    @classmethod
+    def setUpClass(cls):
+        print('**************************************************************')
+        print(cls.__name__)
+        print('**************************************************************')
+
+    def setUp(self):
+        self.project_id = os.environ.get(PROJECT_ID)
+        self.dataset_id = 'drc_ops'
+        self.dataset_ref = DatasetReference(self.project_id, self.dataset_id)
+        self.client = bq.get_client(self.project_id)
+
+        self.schema = [
+            SchemaField("person_id", "INT64"),
+            SchemaField("first_name", "STRING"),
+            SchemaField("last_name", "STRING"),
+        ]
+
+        self.fields = [
+            dict(name='person_id', type='integer', mode='nullable'),
+            dict(name='first_name', type='string', mode='nullable'),
+            dict(name='last_name', type='string', mode='nullable')
+        ]
+
+        self.hpo_id = 'fake_site'
+        self.id_match_table_id = f'drc_identity_match_{self.hpo_id}'
+        self.ps_values_table_id = f'ps_api_values_{self.hpo_id}'
+
+    @mock.patch('utils.bq.get_table_schema')
+    def test_get_case_statements(self, mock_table_schema):
+        # Pre conditions
+        mock_table_schema.return_value = self.schema
+
+        expected = f'\nCASE WHEN first_name IS NULL THEN \'missing_rdr\' ELSE \'missing_ehr\' END AS first_name, , \n' \
+                   f'CASE WHEN last_name IS NULL THEN \'missing_rdr\' ELSE \'missing_ehr\' END AS last_name'
+
+        # Test
+        actual = id_validation.get_case_statements()
+
+        self.assertEqual(actual, expected)
```

Comment:
This test seems to be failing. I think there's an extra comma at the end of the first line of "expected".